pywb/warcserver/inputrequest.py

```diff
@@ -229,8 +229,11 @@ def __init__(self, method, mime, length, stream,
             mime = ''
 
         if mime.startswith('application/x-www-form-urlencoded'):
-            query = to_native_str(query)
-            query = unquote_plus(query)
+            try:
+                query = to_native_str(query)
+                query = unquote_plus(query)
+            except UnicodeDecodeError:
+                query = to_native_str(query, 'iso-8859-1')
```

Comment:
do you think that we should also perform an `unquote_plus` on the `iso` decoded query in order to perform the same operations in the `utf-8` case?

Reply:
Hmmm. I'm having trouble reasoning about this, because at least in the single example I have, we're essentially processing junk (some bytes that never were url-encoded form data), right? I left it out thinking this was a lost cause and `unquote_plus`ed junk was no better than the original; on the other hand I can't imagine it'd be any worse! And I suppose if there's any chance at all that `query` might ACTUALLY be `iso-8859-1` url-encoded form data, then totally, performing an `unquote_plus` sounds like the right move.... I'll make that change momentarily.

Reply:
Pushed! Thanks for taking a look!